modules/virt-attaching-vm-to-primary-udn.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-attaching-vm-to-primary-udn_{context}"]                                  
+= Attaching a virtual machine to the primary user-defined network
+
+You can connect a virtual machine (VM) to the primary user-defined network (UDN) by requesting the pod network attachment, and properly configuring the interface binding.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Edit the `VirtualMachine` manifest to add the UDN interface details, as in the following example:
++
+.Example `VirtualMachine` manifest
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: example-vm
+spec:
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+            - name: udn-l2-net # <1>
+              binding: 
+                name: l2bridge # <2>
+# ...
+      networks:
+      - name: udn-l2-net # <3>
+        pod: {}
+# ...
+----
+<1> The name of the user-defined network interface.
+<2> The name of the binding plugin that is used to connect the interface to the VM. The required value is `l2bridge`.
+<3> The name of the network. This must match the value of the `spec.template.spec.domain.devices.interfaces.name` field.
```

Comment:
And it should also match the `metadata.name` of the `UserDefinedNetwork` resource.